<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_menu" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_widgets" native="true">
        <widget class="WQSpinBox" name="doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>320</y>
           <width>251</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/undo.png</normaloff>:/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/redo.png</normaloff>:/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/copy.png</normaloff>:/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/paste.png</normaloff>:/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/cut.png</normaloff>:/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="toolTip">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A, Ctrl+E</string>
   </property>
  </action>
  <action name="actionUPPERCASE">
   <property name="text">
    <string>UPPERCASE</string>
   </property>
   <property name="toolTip">
    <string>UPPERCASE</string>
   </property>
  </action>
  <action name="actionlowercase">
   <property name="text">
    <string>lowercase</string>
   </property>
   <property name="toolTip">
    <string>lowercase</string>
   </property>
  </action>
  <action name="actionProperCase">
   <property name="text">
    <string>Proper Case</string>
   </property>
   <property name="toolTip">
    <string>Proper Case</string>
   </property>
  </action>
  <action name="actionDefaultTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
   <property name="toolTip">
    <string>Default theme</string>
   </property>
  </action>
  <action name="actionDarkTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
   <property name="toolTip">
    <string>Dark theme</string>
   </property>
  </action>
  <action name="actionLightTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
   <property name="toolTip">
    <string>Light theme</string>
   </property>
  </action>
  <action name="actionModernTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Modern</string>
   </property>
   <property name="toolTip">
    <string>Modern theme</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/find.png</normaloff>:/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="toolTip">
    <string>Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="text">
    <string>Goto</string>
   </property>
   <property name="toolTip">
    <string>Goto line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionMyAction">
   <property name="text">
    <string>My action</string>
   </property>
   <property name="toolTip">
    <string>Some action</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WQSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>wqspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
